lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -140,7 +140,10 @@ sub cache_assets {
             $status = $cache_client->status($asset_request);
         }
         my $msg = "Download of $asset_uri processed";
-        if (my $output = $status->output) { $msg .= ":\n$output" }
+        if (my $output = $status->output) {
```

Comment:
I suggest to move your git commit explanation into the source code as it's not clear to source code users what this is for. An alternative can always be a function that by its name already tells what it tries to achieve

Reply:
Yes, I thought about the lib function with a good name to no need comments. Let's do that

Reply:
How about now?